modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -283,6 +300,12 @@ public ObjectDefinition postObjectDefinition(
 					objectDefinition.getExternalReferenceCode());
 		}
 
+		if (Validator.isNotNull(rootObjectDefinitionExternalReferenceCode)) {
+			_bindObjectDefinitionToRootObjectDefinition(
+				rootObjectDefinitionExternalReferenceCode,
+				serviceBuilderObjectDefinition);
```

Comment:
@G-Sa do we need to reassign the `serviceBuilderObjectDefinition` here (like you did in the PUT logic)?

Reply:
We can use it to ensure more reliability. Good catch!